shopfloor_mobile/static/wms/src/components/packaging-qty-picker.js

```diff
@@ -212,67 +182,95 @@ export var PackagingQtyPickerMixin = {
 
 export var PackagingQtyPicker = Vue.component("packaging-qty-picker", {
     mixins: [PackagingQtyPickerMixin],
-    /** TODO: the trigger has been moved to `updated` because
-     * when if you refresh the same state/page the qty change is not triggered
-     * and the qty stored in the scenario data variable is lost.
-     * (eg: zone_picking/set_line_destination:on_qty_update).
-     * BUT this is still weird because there shouldn't be any need
-     * to retrigger the event if `scan_destination_qty` would not lose its value
-     * when the page is updated.
-     * It seems weird to have to not use `watch` here.
-     * Hence, if we have the time, this is something good to check.
-     */
-    // watch: {
-    //     value: {
-    //         handler: function(newVal, oldVal) {
-    //             this.$root.trigger("qty_edit", this.value);
-    //             console.log("picker trigger");
-    //         },
-    //     },
-    // },
-    mounted: function () {
-        this._init_editable();
+    props: {
+        readonly: Boolean,
+    },
+    data: function () {
+        return {
+            qty_todo: 0,
+            panel: 0, // expand panel by default
+        };
+    },
+    watch: {
+        qty_by_pkg: {
+            deep: true,
+            handler: function () {
+                // prevent watched qty_done to update again qty_by_pkg
+                this.qty_by_pkg_manual = true;
+                this.compute_qty();
+                this.qty_by_pkg_manual = false;
+            },
+        },
     },
     updated: function () {
-        this.$root.trigger("qty_edit", this.value);
+        // Put in updated to prevent triggerring when the component is initialized
+        this.$root.trigger("qty_edit", this.qty_done);
```

Comment:
We also need this on created otherwise the parent component (scenario) is not aware of the right quantity shown on the screen

Reply:
Fixed here https://github.com/OCA/wms/pull/351/files#diff-e9459678436f55fe596b98e51b7c13f5be86268f33c34930cb06fbbad45ce92fR207